Lesson34/selcan/lesson34-app/src/components/Form/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+import { Account } from "../Account";
+
+// TODO: Create a new Component that contains log in form, so that user can type username and password and click submit button to log in.
+// If user is logged in, show Account component.
+// If user is logged out, show Log In component.
+
+export default function Login() {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const [loggedIn, setLoggedIn] = useState("");
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    if (username === "user" && password === "password") {
+      setLoggedIn(true);
```

Comment:
When the form is submitted, you also need to dispatch the login action.
```
dispatch({
        type: UserActionTypes.login,
        payload: {
          isLoggedInUser: true,
          username: "user",
        },
      });
```

Make sure to import UserActionTypes

Reply:
You can actually remove `setLoggedIn(true);` and `const [loggedIn, setLoggedIn] = useState("");` state completely from this component.